config/routes.rb

```diff
@@ -36,7 +36,7 @@ def scoped_navigation_routes(context, navigation)
       end
     end
 
-    resources :documents, controller: :documents do
+    resources :documents, only: [:destroy], controller: :documents do
```

Comment:
This route was, by default, creating all of the restful routes for documents in addition to all of the DocumentNavigation custom routes that were being created in the block. Based on other existing code, the creation of these default routes seemed inadvertent (only a destroy method in controller, later defines only: destroy on line 121)

Reply:
Let's ensure that this is nicely tested

Reply:
Tis